### ISSUE TITLE
Default background color on iOS light theme is wrong

### ISSUE THREAD

**Issue by anagstef** [skwasjer/skwas-cordova-plugin-datetimepicker#41]
**Describe the bug**
After the latest update of the plugin, the color of the Datepicker on iOS (light theme) changed from white to light gray. Is this on purpose?

**Expected behavior**
The default light theme background color should be white (?)

**Plugin version**
v2.1.0

**Device (please complete the following information):**
 - Device: iPhone
 - OS: iOS 13.2

**Comment by skwasjer** [comment]
That is correct. While adding dark mode support I revisited [all colors](https://github.com/skwasjer/skwas-cordova-plugin-datetimepicker/blob/master/src/ios/ModalPickerViewController.m#L176), not just for the dark theme. I aligned the colors to the keyboard popups displayed by the Cordova webview (f.ex. for text, numbers, etc).

I should have documented this better.

I have considered exposing the colors through the plugin options, but just haven't gotten around to this as it's a little more complicated (due to hairline colors, shadows, and other inferred colors).
